config/routes.rb

```diff
@@ -8,14 +7,12 @@
   get '/auth/openstax', as: 'openstax_login'
 
   # User profile route
-  if OpenStax::Accounts.configuration.enable_stubbing?
-    namespace :dev do
-      resources :accounts, only: [:index, :create] do
-        post 'become', on: :member
-        get 'search', on: :collection
-      end
+  namespace :dev do
+    resources :accounts, only: [:index, :create] do
+      post 'become', on: :member
+      get 'search', on: :collection
     end
-  end
+  end if OpenStax::Accounts.configuration.enable_stubbing?
 
```

Comment:
Eh, I think it's clearer to have the `if` be a block vs trailing for multi-line things like this since the indentation makes it clear that it's conditional at the beginning vs catching it at the end after you read it, then having to go back and re-read with the `if` in mind

Reply:
OK I'll switch back